lib/imapoptions

```diff
@@ -1666,16 +1702,24 @@ Blank lines and lines beginning with ``#'' are ignored.
 /* Maximum number of logged in sessions allowed per user,
    zero means no limit */
 
-{ "maxmessagesize", 0, INT, "2.3.17" }
-/* Maximum incoming LMTP message size.  If non-zero, lmtpd will reject
-   messages larger than \fImaxmessagesize\fR bytes.  If set to 0, this
-   will allow messages of any size (the default). */
+{ "maxmessagesize", "0", BYTESIZE, "UNRELEASED" }
+/* Maximum size of messages that will be accepted by Cyrus.  This affects LMTP
+   deliveries, IMAP appends, DAV uploads, etc.  Messages larger than this will
+   be rejected.
```

Comment:
If this can be overwritten per data type using options like `icalendar_max_size`, `webdav_attachments_max_binary_attach_size`, then this shall be spelled explicitly.